src/funding/paylater/config.jsx

```diff
@@ -20,6 +20,17 @@ export function getPaylaterConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
 
+        eligible: ({ experiment, fundingSource }) => {
+            if (
+                experiment
+                && experiment.disablePaylater
+                && !fundingSource  // Exclude standalone buttons
```

Comment:
`fundingSource` can be set in button stack as well.  This isn't exact.  I'm not so sure we can detect this.  @gregjopa Do you know if there is a more reliable way to detect standalone button integration?

Reply:
Ok, I was following @gregjopa advice to use fundingSource to indicate standalone, but I can check if it's vertical or horizontal

Reply:
That won't be enough either.  Will see what we can do.

Reply:
Good point @mnicpt. It this context, `fundingSource` cannot be used to check if its a standalone integration. The style layout may work. Let's dig in and see how we can uniquely identify a standalone integration in this context.

Reply:
Hey @gregjopa after speaking with @mnicpt offline we came to the conslusion that !fundingSource would be ok for this case since that field is only populated if the merchant has set it. Does that align with your understanding?

Reply:
Great to hear you guys paired on this and `!fundingSource` should work! Let's move forward with it.